src/functions.ts

```diff
@@ -3,7 +3,7 @@ import { ModelsAPI } from "./models-api.js";
 
 // defaultModel is the model used for internal calls - for tool calling,
 // or just for chat completions.
-export const defaultModel = "gpt-4o-mini";
```

Comment:
This defaultModel is only used to call _GitHub Models_ - not CAPI. We can totally change it, but I'd opt to leave it as-is since it's cheaper anyway